vendor/src/github.com/docker/engine-api/client/request.go

```diff
@@ -112,7 +112,7 @@ func (cli *Client) sendClientRequest(ctx context.Context, method, path string, q
 			return serverResp, fmt.Errorf("%v.\n* Are you trying to connect to a TLS-enabled daemon without TLS?", err)
 		}
 
-		if cli.transport.Secure() && strings.Contains(err.Error(), "remote error: bad certificate") {
+		if cli.transport.Secure() && strings.Contains(err.Error(), "remote error: tls: bad certificate") {
```

Comment:
same here; that way we can switch Go versions more easily (e.g., in case PPC64LE, or s390x cannot yet switch)